app/controllers/organizations_controller.rb

```diff
@@ -18,6 +18,7 @@ def index
   end
 
   def show
+  @movements = @organization.account.movements.order("created_at DESC").page(params[:page]).per(10)
```

Comment:
Could you please do a `organization.account.movements.order("created_at DESC").page(params[:page]).per(10).explain` in Rails console and paste the generated query here? I think that we can optimize the query.